rust/utils/run-locally/src/invariants.rs

```diff
@@ -15,7 +15,7 @@ pub fn termination_invariants_met(
     solana_config_path: &Path,
 ) -> eyre::Result<bool> {
     let eth_messages_expected = (config.kathy_messages / 2) as u32 * 2;
-    let sol_messages_expected = 1;
+    let sol_messages_expected = 15;
```

Comment:
Maybe worth pulling out into a const that can be used here and in the `for _i in 0..15 {` loop in main.rs?

Reply:
Good point! Since the `MAX_SEQUENCE_RANGE` is 100, we should probably be testing with hundreds rather than tens of messages, or have the sequence range be configurable so we get better cursor logic coverage in the e2e tests.

Reply:
I'm up for that as long as the e2e test run time doesn't change too much